api/v1beta1/ssp_types.go

```diff
@@ -146,7 +146,7 @@ type SSPStatus struct {
 
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
-
+// +kubebuilder:deprecatedversion:warning="ssp.kubevirt.io/v1beta1 ssp is deprecated"
```

Comment:
Is it necesarry to add this explicitly? IIRC this is also the default?

Reply:
Yes, we have to add it, otherwise generator will not add `deprecate: true` attribute.

Reply:
ACK